BigData_ExamineAetrexDB.R

```diff
@@ -0,0 +1,50 @@
+rm(list=ls())
+library(tidyverse)
+library(readxl)
+library(lme4)
+library(patchwork)
+library(effsize)
+
+#-------------------------------------------------------------------------------
+# The purpose of this code is to understand the differences between male and 
+# female foot anthroprometrics
+#-------------------------------------------------------------------------------
+# Loading in the data frame and organizing left and right sides into date frames
+footAnth <- read.csv('C:/Users/eric.honert/Boa Technology Inc/PFL Team - General/BigData/FootScan Data/March23_export.csv')
+colnames(footAnth)[which(names(footAnth) == 'Gender')] <- 'Sex'
+
+footAnth$�..Loc
```

Comment:
Just checking, is this supposed to be "Location"? I'm getting an error on line 18 that "ï==Location" not found